wiki/Help_Centre/de.md

```diff
@@ -0,0 +1,79 @@
+| [Mapping und Editor](/wiki/Help_Centre/Beatmapping) | Deine Beatmaps pflegen, Beatmap-Eigentümerschaft, Beatmap-Slots |
+| [Client](/wiki/Help_Centre/Client) | Fehler und Abstürze, Spielverlauf, Verbindung, Leistung |
+| [Installation dnd Registrierung](/wiki/Help_Centre/Installation_and_registration) | Das Spiel herunterladen, einen Account erstellen |
+| [Store](/wiki/Help_Centre/Store) | Merchandise, osu!go, osu!keyboard, osu!tablet |
+| [Webseite](/wiki/Help_Centre/Website) | Benutzer blockieren, den Support kontaktieren, Seitengestaltung |
+
+## Helfe uns, dir zu helfen
+
+### Log-Dateien
+
+**Log-Dateien sind genaue Aufzeichnungen was der Spiel-Client zu einem bestimmten Zeitpunkt gemacht hat. Wir können sie verwenden, um dein Problem zu lösen.**
+
+Diese Dateien sind extrem praktisch und können sogar das Lösen komplexer Probleme trivial machen.
+
+Wenn ein Mitglied des Supportteams dich nach diesen Logs gefragt hat, dann mache Folgendes:
+
+1. Öffne osu!.
+2. Klicke den `Optionen`-Knopf im Hauptmenü oder drücke `Strg` + `O`.
+3. Tippe `Updatequelle` in die Schnellsuchleiste. Das wird dich zu der aktuellen Updatequelle deines Clients bringen.
+4. Versichere dich, dass es auf `Cutting Edge (Experimentell)` eingestellt ist.
+5. Klicke den Knopf zum Neustarten, der unten auf dem Bildschirm erscheint, sobald du die Updatequelle änderst.
+6. Gehe nochmal zum `Optionen`-Menü und klicke `osu!-Ordner öffnen`.
+7. Finde das `Logs`-Verzeichnis in dem Fenster, welches sich öffnet.
+8. Wähle die benötigte Log-Datei (der Support-Vertreter wird dir sagen, welche) und hänge sie an das Support-Ticket oder Forum-Beitrag.
+
+### Ereignisanzeige
+
+**Die Ereignisanzeige ist eine eingebaute Komponente von Windows, die benutzt werden kann, um Absturzprotokolle zu finden, sollte osu! dir keine geben. Diese Absturzprotokolle können bei der Lösung des Problems helfen.**
+
+Wenn du Probleme mit osu!-Abstürzen hast und osu! dir keine Absturzprotokolle zur Verfügung stellt, dann kannst du sie nur durch die Ereignisanzeige finden.
+
+Wenn ein Mitglied des Supportteams dich fragt, nach Aubsturzprotokollen in der Ereignisanzeige zu suchen, dann mache Folgendes:
+
+1. Nachdem osu! abgestürzt ist, drücke `Win` + `R`, um die Ausführen-Box zu öffnen.
+2. Tippe in der Ausführen-Box `eventvwr` ein und drücke `Enter`. Das wird die Ereignisanzeige starten.
+3. Drücke in der Ereignisanzeige, auf der linken Seite, `Windows-Protokolle` und dann `Anwendung`.
+4. Drücke auf der rechten Seite auf `Aktuelles Protokoll filtern`.
+5. Stelle sicher, dass die `Fehler`-Box im nun öffnenden Fenster einen Haken hat und klicke `OK`.
+6. Drücke `Strg` + `F` und tippe osu! in die Suchleiste. Es wird das erste Absturzprotokoll von osu! finden.
+7. Gehe in den `Details`-Tab und erweitere die `System` und `EventData` durch ein Daraufklicken.
+8. Kopiere den Text dort und füge ihn in den Support-Ticket oder Forum-Beitrag.
+
+### Frame-Zeitdiagram
```

Comment:
I didn't found anything for `Frame Time Graph`. If you have anything, please let me know.

Reply:
I think i wouldnt translate `Frame Time` and keep it as `Frame Time Diagramm` or `Frametime-Diagramm` or something like that

Reply:
it's an osu! feature. if it's not translated in the client, it makes sense to keep it as is

Reply:
yeah, there is no translation in the client. you gotta change it in a few other places, i didn't point it out in my review 😐